docs/assessment/index.md

```diff
@@ -22,11 +22,7 @@ Each assessment is a single directory in the `assessments` folder or any subfold
 
 The assessment ID is the full path relative to `assessments`.
 
-??? note "Format specification for assessment `infoAssessment.json`"
-
-    ```json
-    --8<-- "apps/prairielearn/src/schemas/schemas/infoAssessment.json"
-    ```
+!!! info "[`infoAssessment.json` reference](../schemas/infoAssessment.md) :robot:"
```

Comment:
Not sure the best new text to point to these pages.

Reply:
I'd just write out a sentence, e.g. "See the schema for `infoAssessment.json` for more information about what can be added to this file." or something like that.

Reply:
Copy changed in 3d25c7abf96ccf2b7c4605ad065130bbc2fdeb43.